src/System.Security.Cryptography.Algorithms/tests/DesTests.cs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Encryption.Des.Tests
+{
+
+    public static partial class DesTests
+    {
+        [Fact]
+        public static void DesDefaultCtor()
+        {
+            using (DES des = new DESMinimal())
+            {
+                Assert.Equal(64, des.KeySize);
+                Assert.Equal(64, des.BlockSize);
+                Assert.Equal(CipherMode.CBC, des.Mode);
+                Assert.Equal(PaddingMode.PKCS7, des.Padding);
+            }
+        }
+
+        [Fact]
+        public static void DesIsWeakPositive()
+        {
+            foreach (byte[] key in _knownWeakKeys)
```

Comment:
Consider doing this with a [Theory] and [MemberData] instead.  Then if the asserts fail, the test output will include the failed inputs.